docs/security/access-control.md

```diff
@@ -20,24 +42,60 @@ Gravitino doesn't support metadata authentication. It means that Gravitino won't
 
 :::
 
-
 ## Concept
 
+### Authorization
+
+Gravitino also provides a set of authorization frameworks to interact with different underlying data source
+authorization systems (e.g., MySQL's own access control management and the Apache Ranger access control management system for big data)
+in accordance with its own authorization model and methodology.
+More information you can see the [Authorization push down](authorization-pushdown.md).
+
+### Authentication
+
+As mentioned above, Gravitino uses Ownership to control the rights of securable object in the management category and uses Role to control access securable objects,
+so when a user performs a specific operation on a specified resource,
+Gravitino will perform a composite authentication on the Ownership and Role to which the resource belongs.
+When a user has more than one Role, Gravitino will use the user's current Role for authentication, and the user can switch the current Role to access a different securable object.
+
 ### Role
 
-A metadata object to which privileges can be granted. Roles are in turn assigned to users or groups.
+The traditional rights system generally uses RBAC (Role-Based Access Control) for rights management,
+where each Role contains a collection of different operating privileges for a different securable object.
+When the system adds a new user or user group, you can select the Roles which they are expected to be granted to,
+so that the user can quickly start using it, without waiting for the administrator to gradually set up the access rights to securable object for him.
```

Comment:
```suggestion
so that the user can quickly start using it, without waiting for the administrator to gradually set up the access privileges to securable object for him.
```